### DISCUSSION TITLE
3D Stereoscopic rendering with two camera angles (L+R)

### DISCUSSION THREAD

**Discussion by Byronius7X** [NVlabs/instant-ngp#755] [Ideas / Feature requests]
I've been toying around with getting something like this to happen with modifying a few positional settings in the base_cam.json. the first value after each T, I've been subtracting 0.1 from for the second render to create stereoscopic situation and then joining the two out put render files as a 3D pair in Vegas. They're turning out quite well but a litle wonky here and there. 

I see that the parallax slider doesn't seem to affect the captured camera movement. 
It would be wonderful to have the ability to automatically render two videos out with ffmpeg (one for each eye), perhaps with two generated base_cam.json for stereoscopic output. Perhaps triggered by a switch in the camera window in testbed.

I've attached a sample. You'd be able to use the crossview technique to view it in stereo (in this instance). 

https://user-images.githubusercontent.com/93968012/177916483-070c907b-eb65-4246-94f4-47b93f80cc4c.mp4

**Comment by koktavy** [comment]
For others looking to render Stereoscopic views, see this tool by [_staus](https://twitter.com/_staus)!
https://nerf-pathtools.netlify.app/

**Reply by tufeixp** [reply]
The link is broken, can you fix that?
